Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -475,6 +478,7 @@ void CGContextSynchronize(CGContextRef context) {
     newDrawingState.shadowColor = { 0.f, 0.f, 0.f, 0.f };
     // newDrawingState.blendMode = kCGBlendModeNormal; // TODO GH#1389
     newDrawingState.clippingGeometry = nullptr;
+    newDrawingState.opacityBrush = nullptr;
```

Comment:
These are potentially copied from the base state before them (as comment above.)